### ISSUE TITLE
Add "kubectl explain" support for Application and Release objects

### ISSUE THREAD

**Issue by olbat** [bookingcom/shipper#73]
ATM it does not seems to be possible to get documentation about _Application_ and _Release_ objects using `kubectl explain`:
```
$ kubectl explain application.shipper.booking.com
error: Couldn't find resource for "shipper.booking.com/v1, Kind=Application"

$ kubectl explain release.shipper.booking.com
error: Couldn't find resource for "shipper.booking.com/v1, Kind=Release"
```

Are you planning on adding support for this feature?

**Comment by kanatohodets** [comment]
Hi! Great question. After some quick research, it looks like support for including Custom Resources in the Kubernetes OpenAPI endpoint was merged [just a few days ago](https://github.com/kubernetes/kubernetes/pull/71192). `kubectl explain` queries that OpenAPI endpoint, so once that work is released, you should be able to use `kubectl explain` on Shipper resources without any special work from us.

Judging from the pull request, this work missed the window for Kubernetes 1.14, so hold out for Kubernetes 1.15 :)

**Comment by olbat** [comment]
Awesome, thanks for investigating 👍

**Comment by mingfang** [comment]
FYI This feature was included in the Kubernetes v1.14.0 release.
